images/gptscript/config/main.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. gptscript)."
+  default     = ["gptscript"]
+}
+
+variable "environment" {
+  default = {}
+}
+
+module "accts" {
+  source = "../../../tflib/accts"
+  run-as = 65532
+  uid    = 65532
+  gid    = 65532
+  name   = "nonroot"
```

Comment:
We probably have to run as root to make apk-tools useful :(

Reply:
That's true :/. Will adjust this right now